### PR TITLE
[ZEPPELIN-998] Extend install.md -> Quick Start

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### What is this PR for?
Most of other projects have **Quick Start** or **Getting Started** page for the beginner. Currently, Zeppelin also has [Zeppelin Install](https://zeppelin.apache.org/docs/0.6.0-SNAPSHOT/install/install.html) which is similar with those kind of instruction page. But it has only contents that explain just installation and configuration. So I updated this page to **Quick Start** so that it can include step by step guide for the beginners.

### What type of PR is it?
Improvement & Documentation 

### Todos
* [x] - Add each title link to the head of documentation 
* [x] - Add more information about Zeppelin installation
* [x] - Reorder contents 

### What is the Jira issue?
[ZEPPELIN-998](https://issues.apache.org/jira/browse/ZEPPELIN-998)

### How should this be tested?
See the attached screenshot images

### Screenshots (if appropriate)
screen shot 2016-06-14 at 2 59 47 pm
screen shot 2016-06-14 at 3 00 01 pm
screen shot 2016-06-14 at 3 00 15 pm
screen shot 2016-06-14 at 3 00 27 pm
screen shot 2016-06-14 at 3 00 38 pm

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: docs/install/install.md [hunk 1]
```diff
@@ -19,30 +19,173 @@ limitations under the License.
...
+# If the job exceeds these values, it will be stopped and marked as failed.
+respawn limit 7 5
+
+# zeppelin was installed in /usr/share/zeppelin in this example
+chdir /usr/share/zeppelin
+exec bin/zeppelin-daemon.sh upstart
+```
+
+## What is the next?
+Congratulation to your successful Zeppelin installation! Here are two next steps you might need.
```
Comment (bzz) [inline docs/install/install.md]: I belive it should be `Congratulation _ob_` as it's not an Apache Zeppelin installation that is being congratulated here, but the user.

**Comment by bzz** [comment]
@AhyoungRyu thank you for great effort structuring the docs!

It looks awesome, just two things that I have noticed:
 1. can you please double check that project is referred as Apache Zeppelin everywhere (following branding guidelines)
 2. having `npm` installed is not a requirement to build project from sources

  `mvn package` should be enough and will take care of installing both, local node as well as NPM so I would suggest we remove mention of npm from 'building from source section'

File: docs/install/install.md (hunk 1 above)
Comment (AhyoungRyu) [inline docs/install/install.md]: @bzz Yeah you're right. I just updated it :)

**Comment by AhyoungRyu** [comment]
@bzz Really appreciated to your review again :) 
 - I added `Apache` before the most of `Zeppelin`. 
 - I didn't noticed that "npm, downloaded by Maven during build phase" in [README.md#requirements](https://github.com/apache/zeppelin/blob/master/README.md#requirements). So I removed it as you said.

**Comment by bzz** [comment]
@AhyoungRyu great job! Let me take another pass on it plz.

BTW is removal of `conf/shiro.ini` a part of the quickstart?

File: docs/install/install.md [hunk 2]
```diff
@@ -19,38 +19,113 @@ limitations under the License.
 -->
 {% include JB/setup %}
 
-## Zeppelin Installation
-Welcome to your first trial to explore Zeppelin!
+# Quick Start
+Welcome to your first trial to explore Apache Zeppelin! 
+This page will help you to get started with Apache Zeppelin. Here is the list of this page.
```
Comment (bzz) [inline docs/install/install.md]: This page will help you to get started and here is the list of topics covered

**Comment by AhyoungRyu** [comment]
@bzz Oh my. That's my bad. I'll revert it now :)

File: docs/install/install.md [hunk 3]
```diff
@@ -19,38 +19,113 @@ limitations under the License.
...
+  * [Stop Zeppelin](#stop-zeppelin)
+  * [(Optional) Start Apache Zeppelin with a service manager](#optional-start-apache-zeppelin-with-a-service-manager)
+* [What is the next?](#what-is-the-next)
+* [Apache Zeppelin Configuration](#apache-zeppelin-configuration)
 
-### Install with Binary Package
+## Installation
 
-If you want to install Zeppelin with latest binary package, please visit [this page](http://zeppelin.apache.org/download.html).
+Apache Zeppelin officially tested on the below environment.
```
Comment (bzz) [inline docs/install/install.md]: Apache Zeppelin officially supports and is tested on next environments

File: docs/install/install.md [hunk 4]
```diff
@@ -19,38 +19,113 @@ limitations under the License.
...
+The other option is [building from the source](#building-from-source).
+Although it can be unstable somehow since it is on development status, you can explore newly added feature and change it as you want.
+
+### Downloading Binary Package
+
+If you want to install Apache Zeppelin with a stable binary package, please visit [Apache Zeppelin download Page](http://zeppelin.apache.org/download.html). 
+After unpacking, jump to [Starting Apache Zeppelin with Command Line](#starting-apache-zeppelin-with-command-line) section.
+
+### Building from Source
+If you want to build from the source, there are more requirements.
```
Comment (bzz) [inline docs/install/install.md]: If you want to build from the source, the software below needs to be installed on your system

File: docs/install/install.md [hunk 5]
```diff
@@ -19,38 +19,113 @@ limitations under the License.
...
+    <td>Git</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Maven</td>
+    <td>3.1.x or higher</td>
+  </tr>
+</table>
+
+If you don't have the above requirements yet, please check [Before Build](https://github.com/apache/zeppelin/blob/master/README.md#before-build) section and follow step by step.
```
Comment (bzz) [inline docs/install/install.md]: If you don't have it installed yet, please check... and follow step by step instructions from there.

**Comment by bzz** [comment]
Thanks for prompt response! 
Your documentation update looks great to me, modulo few minor suggestions above.

Please feel free to use them at your will, otherwise I think it's ready to be merged.

**Comment by AhyoungRyu** [comment]
@bzz Thanks for all of your suggestions again. I've just addressed them. Please check my last commit :)

**Comment by bzz** [comment]
Looks great to me, thank you @AhyoungRyu 

Will merge to master, if there is no further discussion

**Pull request closed without merging**